code-security/admin_guide/get-started/connect-your-repositories/add-azurerepos.adoc

```diff
@@ -3,37 +3,84 @@
 [.task]
 == Add Azure Repos to Prisma Cloud Code Security
 
-Integrating Prisma Cloud with Azure Repos makes it possible for Cloud Code Security  to scan your Infrastructure-as-code files (Terraform and CloudFormation) and monitor configuration issues in development.
+Integrating Prisma Cloud with Azure Repos makes it possible for Cloud Code Security to scan your Infrastructure-as-code files (Terraform and CloudFormation) and monitor configuration issues in development.
 As a prerequisite you are required to add the Prisma Cloud IP addresses and hostname for Code Security to an allow list, to https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.html#id7cb1c15c-a2fa-4072-%20b074-063158eeec08[enable access to the Prisma Cloud Console].
+Integration of Azure Repos with Prisma Cloud console uses a Azure DevOps user token for configuration. You can use the user tokens in two ways:
+
```

Comment:
Not sure if we fully understand the complexity here in terms of terminology. 
as you mentioned - a single user token can point single organization (by default, just selecting random Azure repos organization if the user has access to multiple ones) or multiple organizations within Azure repos (if the user is part of multiple organizations).

The real improvement we did here is to support **multiple user tokens** that can be integrated in a single prisma cloud tenant) - before this improvement - authorizing a new token override the existing one

Reply:
@GiladMark why is the support multiple user tokens required? Doesn't the repo get added only once and that is a "single" user's token.? When or why do we need to support multiple user tokens? Is it to allow more than the single user to access the Azure repo directly from our console?